packages/wonder-blocks-modal/components/standard-modal.test.js

```diff
@@ -2,14 +2,78 @@
 import * as React from "react";
 import {shallow} from "enzyme";
 
+import {
+    MediaLayoutContext,
+    MEDIA_DEFAULT_SPEC,
+} from "@khanacademy/wonder-blocks-layout";
 import StandardModal from "./standard-modal.js";
+import {mount, unmountAll} from "../../../utils/testing/mount.js";
 
 describe("StandardModal", () => {
+    beforeEach(() => {
+        unmountAll();
+    });
+
     test("Ensure the ModalCloseButton isn't inside.", () => {
         const wrapper = shallow(
             <StandardModal title="Title" content="Content" footer="Footer" />,
         );
 
         expect(wrapper.find("ModalCloseButton").exists()).toBeFalsy();
     });
+
+    // We already capture desktop snapshots from auto generated tests
+    test("mobile", () => {
+        // Arrange
+        const wrapper = mount(
```

Comment:
This is really where the action is. What do you think about moving it to `// Act`?